app/graph/types/annotator_type.rb

```diff
@@ -1,15 +1,14 @@
-AnnotatorType = GraphQL::ObjectType.define do
-  name 'Annotator'
-  description 'Information about an annotator'
-  interfaces [NodeIdentification.interface]
-  global_id_field :id
```

Comment:
TODO: Verify we no longer need global_id_field :id, and don't need anything to replace it